<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuickGroup</class>
 <widget class="QDialog" name="QuickGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="3">
    <widget class="QPushButton" name="groupButton">
     <property name="text">
      <string>Group</string>
     </property>
     <property name="icon">
      <iconset resource="topo-blend.qrc">
       <normaloff>:/images/group.png</normaloff>:/images/group.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="ungroupButton">
     <property name="text">
      <string>Ungroup</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="clearButton">
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="3" colspan="4">
    <widget class="QListWidget" name="list">
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QListWidget" name="curList"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="topo-blend.qrc"/>
 </resources>
 <connections/>
</ui>
